omnibus/.gitignore

```diff
@@ -9,3 +9,4 @@ cookbooks
 vendor/cookbooks
 build_timestamp
 ldd.out
+Berksfile.lock
```

Comment:
This is actively being updated by `rake dependencies` (the bumpbot is doing this). If we don't want it to be locked, we need to remove the `Berksfile.lock` update here from the top level `tasks/dependencies.rb`. I don't super mind because it's really only for development AFAIK and doesn't get tested.